.hlint.yaml

```diff
@@ -1719,11 +1787,11 @@
     name: "Use 'find' from Relude"
     note: "'find' is already exported from Relude"
     rhs: find
-# - warn:
-#     lhs: Data.Foldable.fold
-#     name: "Use 'fold' from Relude"
-#     note: "'fold' is already exported from Relude"
-#     rhs: fold
+- warn:
```

Comment:
Is there a reason you enabled this warning again?

I use the `recursion-schemes` library heavily in eclair, and they also export `fold`. I think that's why I had to disable it, but I think I use `cata` instead of `fold` everywhere now.

Reply:
There's one place left, should I change it too?

Reply:
No, that's ok. For now just hlint step in CI is fine :+1:.